cmd/thor/flags.go

```diff
@@ -150,6 +150,10 @@ var (
 		Value: "localhost:2112",
 		Usage: "metrics service listening address",
 	}
+	enableDebug = cli.BoolFlag{
```

Comment:
can we add this flag to readme + some info about it there please 😄 maybe we can add some tests around it too